docs/content/configuration/broker.md

```diff
@@ -67,7 +67,6 @@ The broker uses processing configs for nested groupBy queries. And, optionally,
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.query.groupBy.singleThreaded`|Run single threaded group By queries.|false|
```

Comment:
i propose to keep this and add pointer or description of the new context feature. i don't think it is an overlapping with the description you have added at `query-context`.